packages/remirror__extension-callout/src/callout-extension.ts

```diff
@@ -56,6 +88,7 @@ export class CalloutExtension extends NodeExtension<CalloutOptions> {
       attrs: {
         ...extra.defaults(),
         type: { default: defaultType },
+        emoji: { default: '💡' },
```

Comment:
Maybe it's better to allow users to change the default emoji with extension options, like `defaultType` above.